integrations/ingress-nginx/documentation.yaml

```diff
@@ -0,0 +1,88 @@
+exporter_type: included
+app_name_short: Ingress NGINX
+app_name: {{app_name_short}} Controller
+app_site_name: Ingress NGINX
+app_site_url: https://kubernetes.github.io/ingress-nginx
+exporter_name: the Ingress NGINX Controller
+exporter_pkg_name: ingress-nginx
+exporter_repo_url: https://kubernetes.github.io/ingress-nginx/user-guide/monitoring/
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is exposing
```

Comment:
In my experience, we do not need to make changes to ingress nginx. The port 10254 is listening by default, and exposes metrics. This same port is used for healthchecks.

It may or may not be defined explicitly as a `containerPort`, but that does not prevent the port from being accessed by the prometheus scraper.